src/components/Table/Table/Table.module.scss

```diff
@@ -0,0 +1,21 @@
+.table {
+  border-radius: 4px 4px 0 0;
```

Comment:
we have variables for border radius please take a look here: https://style.monday.com/?path=/docs/foundations-round-corners--page 
for example: var(--border-radius-medium);